Source/FakeItEasy.Specs/FakeItEasy.Specs.ruleset

```diff
@@ -13,6 +13,7 @@
     <Rule Id="CA1405" Action="None" />
     <Rule Id="CA1707" Action="None" />
     <Rule Id="CA1709" Action="None" />
+    <Rule Id="CA1822" Action="None" />
     <Rule Id="CA1823" Action="None" />

```

Comment:
It doesn't, but I took a judgement call and decided to loosen the restriction rather than go through and static-ise everything. Having said that, I wouldn't be opposed to tighten everything up to be static wherever possible and re-nable this rule.